src/main/resources/edu/cornell/kfs/ksr/businessobject/datadictionary/SecurityGroupTab.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+	<bean id="SecurityGroupTab" parent="SecurityGroupTab-parentBean" />
+	<bean id="SecurityGroupTab-parentBean" abstract="true" parent="FinancialSystemBusinessObjectEntry"
+          p:actionsProvider-ref="businessObjectActionsProvider"
+          p:boNotesEnabled="true"
```

Comment:
Another spot where the boNotesEnabled flag may need to be removed.

Reply:
removed